### PR TITLE
KTOR-7944 Add kotlinx-browser for htmx plugin

### PR DESCRIPTION
**Pull request by bjhham**
I guess the wasmJs browser target in the multiplatform gradle plugin was implicitly importing kotlinx-browser before, and has since stopped with the latest version.  Either that or I'm going crazy.

### CODE REVIEW

File: src/main/kotlin/io/ktor/plugins/registry/GenerateTestProject.kt [hunk 1]
```diff
@@ -46,6 +47,7 @@ fun main(args: Array<String>) {
     logger.info("Creating project from plugins:\n${pluginIds.joinToString("\n") { " - $it" }}")
     val registryFiles = RegistryOutputFiles(argsMap["build-dir"]!!)
     val latestRelease = registryFiles.ktorReleases.last()
+    val versionProperties = Paths.get("plugins").readVersionProperties()
```
Comment (osipxd) [inline src/main/kotlin/io/ktor/plugins/registry/GenerateTestProject.kt:50]: 💅 
```suggestion
    val versionProperties = Path("plugins").readVersionProperties()
```

**Review by osipxd — APPROVED** [review]

File: src/main/kotlin/io/ktor/plugins/registry/GenerateTestProject.kt (hunk 1 above)
Comment (bjhham) [reply src/main/kotlin/io/ktor/plugins/registry/GenerateTestProject.kt:50]: Needs a different import...

Comment (bjhham) [reply src/main/kotlin/io/ktor/plugins/registry/GenerateTestProject.kt]: Updated

**Review by osipxd — APPROVED** [review]
👍

**Review by e5l — APPROVED** [review]

**Pull request merged**